Controllers/LoginScreen.swift

```diff
@@ -98,7 +101,7 @@ class LoginScreen: UsersDatabase, UITextFieldDelegate {
         self.checkLoggedStatus(token: token)
       }
       if token != nil && (authentificationService?.oauth2.hasUnexpiredAccessToken())! {
-        performSegue(withIdentifier: "SegueFromLogScreenToConnected", sender: self)
+        performSegue(withIdentifier: "ShowInterventionVC", sender: self)
```

Comment:
Segue identifiers should be written in camelCase, with a first letter lowercased